src/main/java/by/itechart/api/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package by.itechart.api.service.impl;
+
+import by.itechart.api.dto.UserDTO;
+import by.itechart.api.entity.User;
+import by.itechart.api.repository.UserRepository;
+import by.itechart.api.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
The same as above.
We already figured out that annotation works fine without this parameter